sphinxcontrib/test_reports/directives/test_common.py

```diff
@@ -6,7 +6,8 @@
 
 from docutils.parsers.rst import Directive
 from sphinx.util import logging
-from sphinx_needs.api import make_hashed_id
+from sphinx_needs.api.need import _make_hashed_id
```

Comment:
I guess we need here some dynamic import, depending on the installed Sphinx-Needs versions.
Otherwise the new release will not be compatible with SPhinx-Needs <4.

something like
```python
try:
    from sphinx_needs.api.need import _make_hashed_id
except ImportErro: < Sphinx-Needs 4
    from sphinx_needs.api import make_hashed_id as _make_hashed_id
```

Reply:
Added already a fix by commit.